cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-simple-cli-guide.adoc

```diff
@@ -0,0 +1,45 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-getting-started-simple-cli-guide"]
+= Tutorial: Simple CLI guide
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-getting-started-simple-cli-guide
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-11-16
+
+This page outlines the minimum list of commands to deploy a ROSA cluster using the command line interface (CLI). 
```

Comment:
Do we need to spell out "Red Hat OpenShift on AWS" at least once on these tutorials?